modules/deleting-the-cluster-logging-operator-by-using-the-cli.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+// * uninstalling/uninstalling-logging.adoc
+
+:_newdoc-version: 2.18.4
+:_template-generated: 2025-07-05
+:_mod-docs-content-type: PROCEDURE
+
+[id="deleting-the-cluster-logging-operator-by-using-the-cli_{context}"]
+= Deleting the {clo} by using the CLI
+
+Delete the {clo} by using the {oc-first}.
+
+.Prerequisites
+
+* You have administrator permissions.
+* You installed the {oc-first}.
+
+.Procedure
+
+. Delete the `ClusterLogForwarder` custom resource (CR) by running the following command:
++
+[source,terminal]
+----
+$ oc delete ClusterLogForwarder -n openshift-logging
```

Comment:
oc delete ClusterLogForwarder  <name> -n openshift-logging.   We need a ClusterLogForwarder name here.

Reply:
done